engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/CreateGameScreen.java

```diff
@@ -138,10 +139,20 @@ public void set(Module value) {
         gameplay.setOptionRenderer(new StringTextRenderer<Module>() {
             @Override
             public String getString(Module value) {
-                return value.getMetadata().getDisplayName().value();
+            	return value.getMetadata().getDisplayName().value();
```

Comment:
Uhh.. you replaced the spaces just before `return` with a tab.

We strictly use 4 spaces over tabs.